include/dirac_quda.h

```diff
@@ -368,9 +383,9 @@ namespace quda {
     QudaMatPCType getMatPCType() const { return matpcType; }
 
     /**
-       @brief  I have no idea what this does
+       @brief returns the number of stencil applications per dslash application; 1 for fused operators, generally 2 otherwise
```

Comment:
I think this description is wrong.  It's 1 for operator that consist of a single hopping term, generally full operators, and 2 for composite operators that consist of two hopping terms.

Reply:
It's now true in practice, however there is an exception with the clover hasenbush operator where the "full" operator is still a separate D_eo and D_oe. Getting this counting correct was necessary for the worker abstraction in various operators.

Reply:
I'll update the wording to make that clearer, though

Reply:
https://github.com/lattice/quda/pull/1488/commits/83d603f91d50b2c48462dc1132aa9a2b100e436e